libscpi/src/meson.build

```diff
@@ -0,0 +1,37 @@
+#  'scpi.g'
+sources = files([
+  'error.c',
+  'expression.c',
+  'fifo.c',
+  'ieee488.c',
+  'lexer.c',
+  'minimal.c',
+  'parser.c',
+  'units.c',
+  'utils.c',
+])
+
+inc_private = include_directories( '.' )
+
+libscpi = both_libraries('scpi',
```

Comment:
Instead of both_libraries, I would generally recommend using library() but setting the meson project option `default_options: ['default_library=both']`.

This allows end users to decide which ones they need, if they really want to. Unless there's a technical reason why the software *requires* that all users have both of them.